lib/index.js

```diff
@@ -156,9 +156,9 @@ $RefParser.resolve = function (path, schema, options, callback) {
  * @returns {Promise}
  * The returned promise resolves with a {@link $Refs} object containing the resolved JSON references
  */
-$RefParser.prototype.resolve = async function (path, schema, options, callback) {
```

Comment:
I think the reason for this might have been for clarity - since there are no typings, it's not easy to understand what `..._args` are supposed to be. 🤷‍♂ not a big deal though, just not sure if this is strictly a good change or not.

Reply:
I assumed the jsdoc annotation should be sufficient, but happy to revert it.

Reply:
Yeah let's revert unless specific reason / need to do it.